src/foam.js

```diff
@@ -4,11 +4,16 @@
  * http://www.apache.org/licenses/LICENSE-2.0
  */
 
-
 (function() {
   var scripts = '';
-
-  var foam = globalThis.foam = Object.assign({
+  function assign(target, source) {
```

Comment:
definitely document why this is needed - prevents overrides in `foam_node` from working, doesn't it? @kgrgreer